public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",

```

Comment:
You can put the details of your project over here.

Reply:
Yes. I'll do it. I need to add favicon as well.